lib/graphlient/errors/execution_error.rb

```diff
@@ -4,12 +4,12 @@ class ExecutionError < Error
       attr_reader :response
 
       def initialize(response)
+        super('the server responded with a GraphQL execution error')
         @response = response
-        super 'the server responded with a GraphQL execution error'
       end
 
       def errors
-        response.data.errors
+        @response.data.errors
```

Comment:
This shouldn't be `@`, we allow overriding things in general, and this would prevent that.